tests/tests_LfD/test_gmm_fitting.py

```diff
@@ -47,19 +58,21 @@ def test_nb_gmm_components(self):
 
     def test_gmr_implementation(self):
 
+        if self._is_arm:
+            return
```

Comment:
can we use if not self._is_arm as in the previous test? :)

Reply:
Or should I change the previous test to a clause guard, as here :)?